src/proto/mod.rs

```diff
@@ -27,9 +29,8 @@ use exonum::{
 };
 use exonum_derive::{BinaryValue, ObjectHash};
 use exonum_proto::ProtobufConvert;
-use failure;
 use protobuf::Message;
-use serde_derive::{Deserialize, Serialize};
+use serde_derive::{Deserialize, Serialize}; // FIXME: remove once `ProtobufConvert` derive is improved (ECR-4316)
```

Comment:
Shouldn't this comment be placed next to `use anyhow as failure`?

Reply:
It seems that is a funny artefact produced by the `rustfmt` bug.